pkg/netconf/nftables.go

```diff
@@ -16,6 +16,7 @@ const (
 	// TplNftables defines the name of the template to render nftables configuration.
 	TplNftables     = "nftrules.tpl"
 	dnsPort         = "domain"
+	dnsProxyZone    = "3"
```

Comment:
We set up additional conntrack zone  specifically for DNS traffic. That's needed because conntrack tracks packet twice, when it's in tenant VRF and when it's in in intenet VRF(where proxy is working). Problem is conntrack doesn't allows duplicate entries in one zone. As for zone 3 -- i think the number was obtained by experiments.